stats/README.md

```diff
@@ -11,6 +11,19 @@ The service consists of 2 parts, a stats calculation library and a transport lay
 
 - Postgresql database for this service
 - Access to Blockscout database
+- (Preferable) Access to Blockscout API
```

Comment:
What does preferable mean and what happens if an access would not be available?

Reply:
I've added some lead to more info, can you recheck and see if it's clear now?

Reply:
Maybe someone on tech call will come up with a better approach 😇